calculators/committee_contribution_list_calculator.rb

```diff
@@ -0,0 +1,50 @@
+class CommitteeContributionListCalculator
+  def initialize(candidates: [], ballot_measures: [], committees: [])
+    @committees = committees
+  end
+
+  def fetch
+    results = ActiveRecord::Base.connection.execute(<<-SQL)
+      -- Schedule A Monetary Contributions
+      SELECT "Filer_ID"::varchar, "Tran_Amt1", "Tran_NamF", "Tran_NamL", "Tran_Date"
+      FROM "efile_COAK_2016_A-Contributions"
+      WHERE "Filer_ID"::varchar IN (#{filer_ids})
+      UNION
+
+      -- Schedule C In-Kind contributions
+      SELECT "Filer_ID"::varchar, "Tran_Amt1", "Tran_NamF", "Tran_NamL", "Tran_Date"
+      FROM "efile_COAK_2016_C-Contributions"
+      WHERE "Filer_ID"::varchar IN (#{filer_ids})
+      UNION
+
+      -- Form 497 Late Contributions
+      -- TODO: we will need to dedupe this against the other forms
```

Comment:
Oh cool. It sounds like `UNION ALL` with some kind of `DISTINCT` on `(Filer_ID, Trans_ID)` is probably the right way to go about removing duplicates though, rather than relying on `UNION`